pkg/infrastructure/kogito_service.go

```diff
@@ -107,7 +107,7 @@ func getServiceEndpoints(client *client.Client, namespace string, serviceHTTPRou
 // CreateKogitoServiceURI provides kogito service URI for given instance name
 func CreateKogitoServiceURI(service v1alpha1.KogitoService) string {
```

Comment:
not related, but, shouldn't this function be changed?
it returns the service url, not uri and does not create it. wdyt?

Reply:
Yep, I agree. What about renaming this to `GetKogitoServiceURL`?

Reply:
I agree with the new naming.

Reply:
Done